bin/public/hardening/__init__.py

```diff
@@ -71,6 +72,7 @@ def harden(args):
     harden_password()
     setup_permissions()
     disable_services()
+    install_proxy_setup()
```

Comment:
I think having a proxy enabled should be optional, there should be property in the general.cfg with something like "http.proxy.enabled" that can be set to false. For simplicity I even think the default should be False. If you want proxy, you enable it.